<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caesar</class>
 <widget class="QWidget" name="Caesar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; font-style:italic;&quot;&gt;Caesar Cipher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="plaintext_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Cantarell&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ciphertext_edit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Cantarell&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encrption_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>96</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>加密-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrption_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>246</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>解密&lt;-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyword_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>密钥：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>350</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
